docker_environment/artifactory-training/scripts/set-internal-hostname.sh

```diff
@@ -6,5 +6,9 @@ curl -m 5 -f -X PATCH \
 -H 'Content-Type: application/yaml' \
 http://localhost:8082/artifactory/api/system/configuration \
 -d '
-urlBase: http://artifactory-training:8082
-'
\ No newline at end of file
+urlBase: http://jfrog-artifactory-training:8082
+localRepositories: #Local repositories configuration
+     conan-tmp: #The local repository name
```

Comment:
Is it really necessary to create a conan-temp repo in this image?

Reply:
Yes